docs/.vuepress/plugins.js

```diff
@@ -1,4 +1,13 @@
 const plugins = [
+  {
+    name: "hardhat-circom",
+    author: "Project Sophon",
+    authorUrl: "https://github.com/projectsophon",
+    url:
+      "https://github.com/projectsophon/hardhat-circom",
```

Comment:
```suggestion
      "https://github.com/projectsophon/hardhat-circom/tree/master",
```

This is annoying, but our script requires a link to the branch.